build.sbt

```diff
@@ -20,7 +24,6 @@ inThisBuild(
         run = Some("sbt ++${{ matrix.scala }} IntegrationTest/test")
       )
     ),
-    crossScalaVersions -= scala211.value,
```

Comment:
zio-sbt doesn't provide support for Scala 2.11 anymore so we don't need this configuration anymore